tests/ttls

```diff
@@ -7,6 +7,12 @@ source "${TESTSSRCDIR}/helpers.sh"
 title PARA "Test SSL_CTX creation"
 $CHECKER "${TESTBLDDIR}/tlsctx"
 
+if [ -n "$ECBASE2URI" ]; then
+    title PARA "Test setting cert/keys on TLS Context"
+    $CHECKER "${TESTBLDDIR}/tlssetkey" "${ECCRT2URI}" "${ECPRI2URI}"
+fi
```

Comment:
would it make sense to add a sanity test with full keys? Either in else branch or just with normal keys? And with RSA keys?

Reply:
RSA keys always have the public exponent so we'd be testing nothing really, not sure we need any other test really, this specific behavior with EC keys is the corner case that needs testing, other fallback behaviors are already tested independently as needed, and the general behavior is also already working.